commons/src/main/java/io/hyscale/commons/exception/HyscaleErrorGroup.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.exception;
+
+public enum HyscaleErrorGroup{
+    SERVICE_SPEC_PROCESSING(12),
```

Comment:
As discussed, need a range of the groups

Reply:
As discussed, for each group range is maintained as 15 starting from 15.